### ISSUE TITLE
Feature request: bulk add to Collections

### ISSUE THREAD

**Issue by jadeqwang** [dwrensha/sandstorm-collections-app#11]
I would like to be able to bulk add a bunch of grains to a Collection from the grain interface, as opposed to only in the Collections interface. Is this possible?

![image](https://cloud.githubusercontent.com/assets/1584904/19362892/31c5c1e0-913d-11e6-96f3-bc25d56946f5.png)

**Comment by ocdtrekkie** [comment]
This might be hard in the terms that Collections doesn't have any special status in Sandstorm above any other app, and you'd also need a box to select which Collection to add it to.

The easiest approach here would probably to, inside the Collection, create a UI to allow selecting multiple grains at once to add. (There might be some complexities with this though, different grains might need to be added with different permissions, for instance, so you'd end up with a big permissions table to fill out.)

**Comment by zenhack** [comment]
We might be able to do something where we trigger a powerbox offer for the selected set of grains -- which a collection could advertise support for. The UX would be weird though, since Sandstorm wouldn't know A-priori what was going to accept the offer, so labelling the button "add to Collection" would be confusing if the powerbox then suggested non-collection choices. But maybe we could do a generic share/export button...

**Comment by ocdtrekkie** [comment]
Yeah, I was also thinking about that, though I feel like if we did that, people would expect a lot of additional functionality there.

**Comment by zenhack** [comment]
...I suppose we could fold the mass-backup functionality into that. I've been vaguely thinking of wiring up the backup stuff to the powerbox anyway.
